tools/agents/quota-lib.ps1

```
@@ -57,3 +57,63 @@ function Test-QuotaError([string]$Text, [string]$Agent) {
+    # מחזיר @{allowed; used; cap; warn; reason}. רישום לפני אישור.
+    $cap = Get-AgentCap $Agent $Model
+    $s = Read-AgentState
+    $key = "spend_${Agent}_${Model}"
+    $win = Get-SpendWindowId $Agent
+    $cur = $s.PSObject.Properties[$key]
+    $used = 0
+    if ($null -ne $cur -and $cur.Value.window -eq $win) { $used = [int]$cur.Value.count }
+
+    if ($null -ne $cap -and ($used + $Count) -gt $cap) {
+        # הקריאה לא תצא — וגם נרשם קירור עד האיפוס, שידלק מעצמו.
+        Set-AgentCooldown $Agent (Get-NextGeminiReset) "תקרה יומית: $used/$cap" | Out-Null
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Set-AgentCooldown' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)